packages/blockchain-link-types/src/params.ts

```diff
@@ -56,4 +56,5 @@ export interface AccountInfoParams {
         ledger: number;
         seq: number;
     };
+    tokenAccounts?: string[]; // solana only, token accounts to fetch txids for
```

Comment:
wdyt about smthng like
```suggestion
    tokenAccountsPubKeys?: string[]; // solana only, token accounts to fetch txids for
```